variant-switcher/src/ui.tsx

```diff
@@ -15,7 +15,7 @@ const App: React.FC = () => {
     const [propertyName, setPropertyName] = React.useState('Theme');
     const [fromVariant, setFromVariant] = React.useState('Light');
     const [toVariant, setToVariant] = React.useState('Dark');
-    const [deepSwitch, setDeepSwitch] = React.useState<'true' | 'false'>('true');
+    const [deepSwitch, setDeepSwitch] = React.useState<'true' | 'false'>('false');
```

Comment:
we're not leaving the default behavior the same?

Reply:
> Logic here is that ppl who built their component perfectly should not be affected; ppl who used light theme children to build dark theme parents would prefer the deep switch to be disabled.

So I think that since new users might find `false` more useful, and old users who are already building components rigidly wouldn't see a difference between `false` and `true`, I am inclined to leave it `false`